planner/mapper/mapper.go

```diff
@@ -267,7 +267,7 @@ func resolveAggregates(
 	inputFields []Requestable,
 	mapping *core.DocumentMapping,
 	collectionName string,
-	schema client.SchemaDescription,
+	schema client.CollectionDefinition,
```

Comment:
Islam (https://github.com/sourcenetwork/defradb/pull/2339#discussion_r1503974819):

todo: rename the param to colDef or something similar please

Reply:
My bad, thanks for spotting :)

- [ ] Rename param